proj/src/main/webapp/static/chat.html

```diff
@@ -51,35 +59,140 @@
+                            <p class="text-left"><strong>tags</strong></p>
+                        </div>
+                        <div>
+                            <p id="user-tags" class="text-left"></p>
+                        </div>
+                    </li>
             </div>
-
-            <div class="col" id="chat-screen"> 
-                <h1 id="big-title"> Chat Room </h1>
-                <div class="chat-and-input"> 
+            </div>
+                
+                <!-- settings drop down-->
+                
+                <a href="#settings-submenu" data-toggle="collapse" aria-expanded="false" class="list-group-item list-group-item-action flex-column align-items-start">
+                    <div class="d-flex w-100 justify-content-start align-items-center">
+                        <span class="fa fa-user fa-fw mr-3 fa-1x"></span>
+                        <span class="menu-collapsed"><strong>settings</strong></span>
+                        <span class="submenu-icon ml-auto"></span>
+                    </div>
+                </a>
+                <div id='settings-submenu' class="collapse sidebar-submenu">
+                    <a href="#" class="bg-light bg-grey list-group-item list-group-item-action">
+                        <span class="menu-collapsed">edit profile</span>
+                    </a>
+                    <a href="#" class="bg-light bg-grey list-group-item list-group-item-action">
+                        <span class="menu-collapsed">change password</span>
+                    </a>
+                    <a class="bg-light bg-grey list-group-item list-group-item-action" id="btnLogout">
+                        <span class="menu-collapsed">sign out</span>
+                    </a>
+                </div> 
+                
+                <!-- end of scroll but not the list-->
+                <!--</div>-->
+                <!-- Separator-->
+                <li class="list-group-item sidebar-separator menu-collapsed"></li>
+                <!-- left bottom anchor-->
+                <div class="bottom">
+                    <a href="#top" data-toggle="sidebar-colapse" class="bg-light bg-gold list-group-item list-group-item-action d-flex align-items-center">
+                        <div class="d-flex w-100 justify-content-start align-items-center">
+                            <span id="collapse-icon" class="fa fa-2x mr-3"></span>
+                            <span id="collapse-text" class="menu-collapsed">Collapse</span>
+                        </div>
+                    </a>
+                </div>
+            </ul>
+        </div>
+        <!-- main chat section -->
+        <div class="col p-4">
+            <div class="card" id="main-chat">
+                <h1 class="display-4 card-header" id="big-title"> Chat Room </h1>
+                <!-- chat display-->
+                <div class="card-body" id="message-list">
+                    <!-- template for later use -->
```

Comment:
Can you mark this with a TODO so it is easily found as something that needs more work in the future. Also what will this be used as a template for?

Reply:
yup and the template is for the general each message with a time stamp on the upper right corner